packages/cf-component-card/src/CardDrawers.js

```diff
@@ -9,23 +9,40 @@ let UNIQUE_ID = 0;
 class CardDrawers extends React.Component {
   constructor(props) {
     super(props);
+
     this._cardId = UNIQUE_ID++;
+
+    this.state = {
+      active: null
+    };
+  }
+
+  handleLinkClick(id) {
+    this.props.onClick && this.props.onClick();
+
+    this.setState(state => ({
+      active: state.active === id ? null : id
+    }));
   }
 
   render() {
+    const active = this.props.hasOwnProperty('active')
```

Comment:
```jsx
<CardDrawers active={undefined}>
```
results in

```js
this.props.hasOwnProperty('active') === true
```

right? It's probably better to test if `this.props.active` is `undefined` instead.